Shared/ForChildren/Pexe/PexesoViewModel.swift

```diff
@@ -92,7 +92,12 @@ class PexesoViewModel: PexesoViewModeling {
 
     func select(phrase: PexesoContent) {
         guard canRotate else { return }
-        guard !phrase.selected else { return }
+        guard !phrase.selected else {
+            if case .won = state {
+                soundService.play(path: phrase.translation.soundFileName)
```

Comment:
Once you won the game, this line could trigger the validation function every tap. Please consider adding one more if statement for that.